docs/docs/02-apps/02-interchain-accounts/09-active-channels.md

```diff
@@ -7,7 +7,9 @@ slug: /apps/interchain-accounts/active-channels
 
 # Understanding Active Channels
 
-The Interchain Accounts module uses [ORDERED channels](https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics#ordering) to maintain the order of transactions when sending packets from a controller to a host chain. A limitation when using ORDERED channels is that when a packet times out the channel will be closed.
+The Interchain Accounts module uses either [ORDERED or UNORDERED](https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics#ordering) channels to maintain the order of transactions when sending packets from a controller to a host chain. 
```

Comment:
> to maintain the order of transactions when sending packets from a controller to a host chain

I don't think this part is necessarily true anymore when using `Unordered` channels, right? If using `Unordered` no guarantees or assumptions can be made wrt ordering on the host chain.